sql/src/test/java/io/crate/execution/jobs/CountContextTest.java

```diff
@@ -45,6 +46,14 @@
 
 public class CountContextTest extends CrateUnitTest {
 
+    private static CountPhase countPhaseWithId(int phaseId) {
+        CountPhase phase = mock(CountPhase.class);
```

Comment:
Could we use the regular constructor here? Mocking simple objects like a CountPhase doesn't seem right to me. It's almost like `mock(List.class)`

Reply:
Right, the CountPhase is quite simple.